Sources/Basics/HTTPClient.swift

```diff
@@ -0,0 +1,262 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Foundation
+#if canImport(FoundationNetworking)
+// TODO: this brings OpenSSL dependency` on Linux
+// need to decide how to best deal with that
```

Comment:
BTW, the TSC Downloader already brings that same dependency today.

Reply:
https://github.com/apple/swift-package-manager/pull/3040#issuecomment-724951369